udev/70-printers.rules

```diff
@@ -1,4 +1,4 @@
 # Low-level USB device add trigger
 ACTION=="add", SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ENV{ID_USB_INTERFACES}=="*:0701??:*", TAG+="systemd", ENV{SYSTEMD_WANTS}="configure-printer@usb-$env{BUSNUM}-$env{DEVNUM}.service"
 # Low-level USB device remove trigger
-ACTION=="remove", SUBSYSTEM=="usb", ENV{ID_USB_INTERFACES}=="*:0701*:*", RUN+="udev-configure-printer remove %p"
+ACTION=="remove", SUBSYSTEM=="usb", ENV{INTERFACE}=="7/1/*", ENV{INTERFACE}!="7/1/4", RUN+="udev-configure-printer remove %p"
```

Comment:
Would you mind explaining about values of env variable ```INTERFACE```? Why value ```7/1/4``` is omitted? It would be good to have a brief comment in the code too.

Reply:
seems  this is a trick from ubuntu, because all of the ippusbxd protocol (7/1/4) devices has been taken care by another rules, I think let's just leave this file as the original status, I will revert this one.

Reply:
I will still keep ENV{INTERFACE}=="7/1/*" in this file, due to in systemd [0],  it goes to default_end directly when removing action occurred, which meant __IMPORT{builtin}="usb_id"__ will not be executed, so we have to use ENV{INTERFACE} when removing usb printer.



[0] https://github.com/systemd/systemd/blob/master/rules.d/50-udev-default.rules.in